src/components/DownloadAndShareComponents/hooks/useResourceFormSubmit.ts

```diff
@@ -60,30 +60,31 @@ const useResourceFormSubmit = (props: UseResourceFormProps) => {
       setTermsInLocalStorage(terms);
     }
     if (props.type === "download") {
-      const downloadsPayload = getHubspotDownloadsFormPayload({
-        hutk,
-        data: {
-          ...data,
-          ...utmParams,
-          oakUserId: posthogDistinctId ? posthogDistinctId : undefined,
-          schoolName:
-            schoolId === "homeschool" || schoolId === "notListed"
-              ? schoolId
-              : schoolName,
-        },
-      });
-      const hubspotFormResponse = await hubspotSubmitForm({
-        hubspotFormId: hubspotDownloadsFormId,
-        payload: downloadsPayload,
-      });
-
       await downloadLessonResources(
         slug,
         downloads as DownloadResourceType[],
         props.isLegacyDownload,
       );
-      return hubspotFormResponse;
     }
+
+    const downloadsPayload = getHubspotDownloadsFormPayload({
+      hutk,
+      data: {
+        ...data,
+        ...utmParams,
+        oakUserId: posthogDistinctId ? posthogDistinctId : undefined,
+        schoolName:
+          schoolId === "homeschool" || schoolId === "notListed"
+            ? schoolId
+            : schoolName,
+      },
+    });
+    const hubspotFormResponse = await hubspotSubmitForm({
+      hubspotFormId: hubspotDownloadsFormId,
```

Comment:
does share go to the same form as downloads?

Reply:
Yeah i questioned this too, but yes it uses the same form on HubSpot.